gips/datahandler/torque.py

```diff
@@ -0,0 +1,74 @@
+"""Torque-based backend for the GIPS work scheduler."""
+
+# originally copied from:
+# https://raw.githubusercontent.com/jdherman/hpc-submission-scripts/master/submit-pbs-loop.py
+
+from subprocess import Popen, PIPE
+
+from gips import utils
+
+pbs_directives = [
+    # for meaning of directives see
+    #   http://docs.adaptivecomputing.com/torque/4-0-2/Content/topics/commands/qsub.htm
```

Comment:
Thanks for the suggestion; now that I look at `man qsub` I can clearly see the directives in there.